### PR TITLE
Add noembed setup scripts

### PR DESCRIPTION
**Pull request by foolip**
Tested on a fresh Debian VM and appearing to do the right thing, but
not tested end-to-end because of certs.

Fixes https://github.com/whatwg/misc-server/issues/52.

### CODE REVIEW

File: debian/noembed/04-participate [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+git clone https://github.com/whatwg/participate.whatwg.org.git "$PARTICIPATE_DIR"
+chown -R noderunner:noderunner "$PARTICIPATE_DIR"
+
+cd "$PARTICIPATE_DIR"
+sudo -u noderunner npm install
+
+read -p "Copy private-config.sample.json to private-config.json and fill in all the values appropriately. Then press enter to continue"
+
+echo "@reboot cd $PARTICIPATE_DIR && pm2 start -i 4 --name="participate" npm -- start" | sudo -u noderunner crontab -
```
Comment (domenic) [inline debian/noembed/04-participate]: Do these quotes need to be escaped?

Comment (domenic) [inline debian/noembed/04-participate]: Also, any thoughts as to whether this is good enough, or if https://futurestud.io/tutorials/pm2-restart-processes-after-system-reboot would be better?

**Comment by domenic** [comment]
Note to future self: when we move build.whatwg.org here, we should use https://futurestud.io/tutorials/pm2-advanced-app-configuration-with-json-file to configure multiple apps.

File: debian/noembed/04-participate (hunk 1 above)
Comment (foolip) [reply debian/noembed/04-participate]: Oh, by good fortune this ended up writing `@reboot cd  && pm2 start -i 4 --name=participate npm -- start` to crontab instead, and that happens to work. I'll just remove the quotes.

Comment (foolip) [reply debian/noembed/04-participate]: OK, looks like `pm2 startup systemd` should get us hooked up with the init system properly. I think that makes sense, and will file a follow-up issue.

**Pull request merged**